docs/disease.rst

```diff
@@ -58,7 +58,7 @@ disease_stage
 ~~~~~~~~~~~
 
 This attribute is used to describe the stage of disease. If the disease is a cancer, this attribute describes
-the extent of cancer development, typically including an AJCC stage group (i.e., Stage 0, I-IV), though other staging
+the extent of cancer development, typically including an AJCC (i.e., Stage 0, I-IV), though other staging
```

Comment:
why remove the 'stage group'? AJCC is the acronym of the American Joint Committee on Cancer.

Reply:
@julesjacobsen Apologies, this is an error! I checked the ISO doc and it's still present. A commenter suggested to add TNM to this sentence, which I did add, but then later removed because of the TNM_finding field below. I may have accidently removed 'stage group' with it. Will fix this.